client/src/App.js

```diff
@@ -25,6 +25,10 @@ function App() {
   }, [dispatch]);
   console.log(state);
 
+  // Maybe we should wrap it this way?
+  // if (state.loading) {
+  //   return "Loading";
+  // } else { }
```

Comment:
We can do this as a possible first iteration of a loading state - instead of returning "Loading", you probably want to return a CircularProgress (material-ui component) that is centered